src/xmipp/libraries/py_xmipp/deepLearningToolkitUtils/utils.py

```diff
@@ -0,0 +1,52 @@
+import os
+import traceback
+
+BAD_IMPORT_MSG='''
+Error, tensorflow/keras is probably not installed. Install it with:\n  ./scipion installb deepLearningToolkit
+If gpu version of tensorflow desired, install cuda 8.0 or cuda 9.0
```

Comment:
I'm not very happy about this. We should solve dependencies at the install time.
Also this is xmipp code, so we should be able to do it independently form scipion

Reply:
let me know if you have fixed